### PR TITLE
Add VS Code launch config for debug gem

### PR DESCRIPTION
**Pull request by andyw8**
Closes https://github.com/Shopify/ruby-dev-exp-issues/issues/748

### Type of Change

- [ ] Add RBI for a new gem
- [ ] Modify RBI for an existing gem
- [x] Other: 

### Changes

Add launch configuration to support debugging through VS Code.

### CODE REVIEW

**Comment by andyw8** [comment]
Not sure why this doesn't work... the command in the output looks correct:

`rdbg --command --open --stop-at-load -- bin/test /Users/andyw8/src/github.com/Shopify/rbi-central/gem/test/rbi-central/gem_test.rb`

But it fails with `Couldn't start debug session. The debuggee process exited with code 0` as if it's skipping any breakpoints that I add through the UI.

**Comment by andyw8** [comment]
(FYI @st0012, I'll keep investigating but maybe you can see something obvious).

File: gem/rbi-central.gemspec [hunk 1]
```diff
@@ -32,6 +32,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency("spoom", ">= 1.1.12")
   spec.add_dependency("tapioca", ">= 0.9.2")
   spec.add_dependency("thor", ">= 1.2.1")
+  spec.add_development_dependency("debug")
```
Comment (Morriar) [inline gem/rbi-central.gemspec:35]: Same, I think this only makes sense inside `gem/`

Comment (andyw8) [reply gem/rbi-central.gemspec:35]: Do you mean in `gem/Gemfile` rather than the `gemspec`?

Comment (Morriar) [reply gem/rbi-central.gemspec]: My bad I didn't read the filename properly 🤦 

Here's good 👍

**Pull request merged**